.github/workflows/release.yml

```diff
@@ -0,0 +1,40 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  Release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2 # https://github.com/actions/checkout
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v1 # https://github.com/actions/setup-node
+        with:
+          node-version: 12
+
+      - name: Install
+        run: yarn --offline
+
+      - name: Build
+        run: yarn build
+
+      - name: Continuous integration check
+        run: yarn ci-check
+
+      - name: Release
+
+        # uses: actions/create-release@latest # https://github.com/actions/create-release
+
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: echo 'release'
```

Comment:
These will be swapped out for the commented-out code in this step before merging - it will create the release in GitHub based on the tag pushed from https://github.com/carbon-design-system/ibm-cloud-cognitive/pull/164/files#r514282150